gamma_bridge/exceptions.py

```diff
@@ -40,3 +40,13 @@ class NotBaseStorageError(GammaValueError):
     def __init__(self, message=None, *args):
         self.message = "The value type must be BaseStorage. {}".format(message)
         super(GammaException, self).__init__(self.message, *args)
+
+
+class GammaEventDataError(GammaException):
+    """
+    Error is occurred when tracking event don't contain data, required to build uid
```

Comment:
Add dot at the end please.

Reply:
Also typo in don't.
Should be `Error is occurred when tracking event doesn't contain data, required to build uid`.
Also keep in mind that docstring can't exceed 80 chars.

Reply:
done